packages/sdk-codegen/src/codeGen.ts

```diff
@@ -791,11 +802,17 @@ export abstract class CodeGen implements ICodeGen {
    * @param type that receives assignments
    * @param inputs to assign to type
    */
-  assignType(indent: string, type: IType, inputs: ArgValues): string {
+  assignType(
+    indent: string,
+    type: IType,
+    mapped_type: IMappedType | null,
+    inputs: ArgValues
+  ): string {
     const args: string[] = []
     const bump = this.bumper(indent)
     Object.values(type.properties).forEach((p) => {
-      let v = this.argValue(bump, p, inputs)
+      // let v = this.argValue(bump, p, inputs)
+      let v = this.argValue(indent, p, inputs)
```

Comment:
note to self: re-do TS indentation/openers and remove special casing indentation